misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/PathAssertionsSpec.kt

```diff
@@ -34,10 +34,11 @@ abstract class PathAssertionsSpec(
     fun describeFun(vararg funName: String, body: Suite.() -> Unit) =
         describeFunTemplate(describePrefix, funName, body = body)
 
-    describeFun(exists.name) {
+    describeFun(exists.name, existsNot.name) {
         val existsFun = exists.lambda
-        context("non existing") {
-            it("throws an AssertionError") {
+        val existsNotFun = existsNot.lambda
+        context("file or folder exists") {
+            it("${exists.name} throws an AssertionError when file does not exist") {
```

Comment:
this contradicts the given context which states the `file or folder exists`. move this to `file or folder does not exist`
Moreover I would introduce two context, one for `file exists` and one for `folder exists` (same same but different for `file does not exist` and `folder does not exist`)